README.md

```diff
@@ -0,0 +1,53 @@
+# Flatpak for DOSBox
+
+## Installation
+
+This Flatpak is available on
+[Flathub](https://flathub.org/apps/details/com.dosbox.DOSBox).
+After following the [Flatpak setup guide](https://flatpak.org/setup/),
+you can install it by entering the following command in a terminal:
+
+```bash
+flatpak install --user flathub com.dosbox.DOSBox -y
+```
+
+Once the Flatpak is installed, you can run DOSBox using your desktop environment's
+application launcher.
+
+## Updating
+
+This Flatpak follows the latest stable DOSBox version.
```

Comment:
I am not sure if this is actually useful thing to have in README. Most Flatpak users either use GUI (GNOME Software, KDE Discover, mintinstall etc.) or already know the actual commands.

Reply:
Maybe… I think the README should be as self-sufficient as possible. Otherwise, it would only document limitations and build instructions.

We should probably "solve" this on flathub.org itself instead, so we don't have to copy this in every Flatpak repository.

Reply:
> We should probably "solve" (i.e. document) this on flathub.org itself instead, so we don't have to copy this in every Flatpak repository.

Yeah, I agree. Not many users actually read READMEs in Git repositories.